resource_booking/models/res_partner.py

```diff
@@ -0,0 +1,26 @@
+from odoo import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    resource_booking_count = fields.Integer(
+        compute="_compute_resource_booking_count", string="Resource booking count"
+    )
+    resource_booking_ids = fields.One2many(
+        "resource.booking", "partner_id", string="Bookings"
```

Comment:
This one is incorrect, as it only reflects the first attendee. It should be a computed one.

Reply:
Change to many2many field + default_partner_ids.